Deprecate adding two Time instances with ActiveSupport::TimeWithZone (#52343)
rails/rails: activesupport/CHANGELOG.md

*   Deprecate addition for `Time` instances with `ActiveSupport::TimeWithZone`.

>     Currently adding time instances together such as `10.days.ago + 10.days.ago` produces a nonsensical future date. This behavior is deprecated and will be removed in Rails 8.0.

Reviewer: (non-blocking)

```suggestion
    Previously adding time instances together such as `10.days.ago + 10.days.ago` produced a nonsensical future date. This behavior is deprecated and will be removed in Rails 8.0.
```

> *   Deprecate adding two Time instances with ActiveSupport::TimeWithZone
>     Right now adding two time instances together such as `10.days.ago + 10.days.ago` produces a nonsensical future date.
>     Includes a deprecation to remove this behaviour in Rails 8.0
> 
>     *Nick Schwaderer*

Reviewer 2: I think this reads better and is clearer like this:

```suggestion
*   Deprecate addition for `Time` instances with `ActiveSupport::TimeWithZone`.

    Currently adding time instances together such as `10.days.ago + 10.days.ago` produces a nonsensical future date. This behavior is deprecated and will be removed in Rails 8.0.

    *Nick Schwaderer*
```
